src/cli/one_helper/oneform_helper.rb

```diff
@@ -0,0 +1,256 @@
+                File.basename(d[:source])
+            end
+
+            column :DEPLOYMENTS, 'DEPLOYMENTS', :left, :expand => true do |d|
+                if d[:deployment_confs].nil? || d[:deployment_confs].empty?
+                    '--'
+                else
+                    d[:deployment_confs].map {|conf| conf[:inventory] || '--' }.join(', ')
+                end
+            end
+
+            column :STATE, 'Group', :left, :size => 10 do |d|
+                d[:state]
+            end
+
+            default :NAME, :DEPLOYMENTS, :STATE
+        end
+    end
+
+    # List drivers pool
+    #
+    # @param client  [OneForm::Client] Petition client
+    # @param options [Hash]            CLI options
+    def list_driver_pool(client, options)
+        response = client.list_drivers
+
+        if CloudClient.is_error?(response)
+            [response[:err_code], response[:message]]
+        else
+            if options[:json]
+                [0, JSON.pretty_generate(response)]
+            elsif options[:yaml]
+                [0, response.to_yaml(:indent => 4)]
+            else
+                table = format_driver_pool
+
+                table.show(response, options)
+                table.describe_columns if options[:describe]
+
+                0
+            end
+        end
+    end
+
+    # List driver pool continiously
+    #
+    # @param client  [OneForm::Client] Petition client
+    # @param options [Hash]            CLI options
+    def top_driver_pool(client, options)
+        options[:delay] ? delay = options[:delay] : delay = 4
+
+        begin
+            loop do
+                CLIHelper.scr_cls
+                CLIHelper.scr_move(0, 0)
+
+                list_driver_pool(client, options)
+
+                sleep delay
+            end
+        rescue StandardError => e
+            STDERR.puts e.message
+            exit(-1)
+        end
+
+        0
+    end
+
+    # Show driver detailed information
+    #
+    # @param client           [OneForm::Client] Petition client
+    # @param driver_name         [String]          driver name
+    # @param options          [Hash]            CLI options
+    def format_resource(client, driver_name, options)
+        response = client.get_driver(driver_name)
+
+        if CloudClient.is_error?(response)
+            [response[:err_code], response[:message]]
+        else
+            if options[:json]
+                [0, JSON.pretty_generate(response)]
+            elsif options[:yaml]
+                [0, response.to_yaml(:indent => 4)]
+            else
+                str    = '%-20s: %-20s'
+                str_h1 = '%-80s'
+
+                CLIHelper.print_header(
+                    str_h1 % "#{File.basename(response[:source]).upcase} ONEFORM DRIVER INFORMATION"
+                )
+
+                puts Kernel.format str, 'NAME', response[:name]
+                puts Kernel.format str, 'DESCRIPTION', response[:description]
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against an allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html